### PR TITLE
Add onnx ops

### PR DESCRIPTION
**Pull request by tomkoren21**

### CODE REVIEW

File: onnx2kerastl/operation_layers.py [hunk 1]
```diff
@@ -223,6 +223,52 @@ def target_layer(x, axis=params['axes'], keepdims=params['keepdims']):
...
+    if params.get("axes") is not None: #opset 13
+        axes = params.get("axes")
+    elif len(node.input) == 2:
+        axes = layers.get(node.input[1])
+    noop_with_empty_axes = bool(params.get("noop_with_empty_axes", False))
+    keepdims = params.get("keepdims", True)
+    if not noop_with_empty_axes:
+        layers[node_name] = tf.math.reduce_min(layers[node.input[0]], axis=axes, keepdims=keepdims)
+    else:
+        layers[node_name] = layers[node.input[0]]
```
Comment (doronHarnoy) [inline onnx2kerastl/operation_layers.py:246]: > Defines behavior if 'axes' is empty. Default behavior with 'false' is to reduce all axes. When axes is empty and this attribute is set to true, input tensor will not be reduced,and the output tensor would be equivalent to input tensor.

from my look at the documentation the condition should be `not noop_with_empty_axes and axes is None`

Comment (doronHarnoy) [reply onnx2kerastl/operation_layers.py:246]: Can you also please convert the if condition from `not noop_with_empty_axes` to `if noop_with_empty_axes`, It feels confusing 😄

File: onnx2kerastl/operation_layers.py [hunk 2]
```diff
@@ -223,6 +223,52 @@ def target_layer(x, axis=params['axes'], keepdims=params['keepdims']):
...
+    :param keras_name: resulting layer name
+    :return: None
+    """
+    if params.get("axes") is not None: #opset 13
+        axes = params.get("axes")
+    elif len(node.input) == 2:
+        axes = layers.get(node.input[1])
+    noop_with_empty_axes = bool(params.get("noop_with_empty_axes", False))
+    keepdims = params.get("keepdims", True)
+    if not noop_with_empty_axes:
```
Comment (doronHarnoy) [inline onnx2kerastl/operation_layers.py:266]: same here

File: onnx2kerastl/operation_layers.py [hunk 3]
```diff
@@ -585,3 +639,24 @@ def convert_trilu(node, params, layers, lambda_func, node_name, keras_name):
...
+
+def convert_cumsum(node, params, layers, lambda_func, node_name, keras_name):
+    exclusive = bool(params.get("exclusive", 0))
+    reverse = bool(params.get("reverse", 0))
+    layers[node_name] = tf.math.cumsum(layers[node.input[0]], layers[node.input[1]],
+                                       exclusive=exclusive, reverse=reverse)
+
+
+def convert_is_inf(node, params, layers, lambda_func, node_name, keras_name):
+    if params.get("detect_negative") is not None or params.get("detect_negative") is not None:
```
Comment (doronHarnoy) [inline onnx2kerastl/operation_layers.py:652]: what is the difference here between the two conditions? maybe you wanted `detect_positive` there?

File: onnx2kerastl/operation_layers.py [hunk 4]
```diff
@@ -585,3 +639,24 @@ def convert_trilu(node, params, layers, lambda_func, node_name, keras_name):
...
+        raise AttributeError("Unsupported params detected in isInf conversion: detect_negative/detect_positive")
```
Comment (doronHarnoy) [inline onnx2kerastl/operation_layers.py:653]: I think we can detect if the inf is negative or positive (maybe just checking if bigger than 0) so we don't have to throw an exception here

File: test/layers/elementwise/test_round.py [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+    def __init__(self):
+        super(TorchRound, self).__init__()
+
+    def forward(self, x):
+        return torch.round(x)
+
+
+def test_round():
+    pt_model = TorchRound()
+    error = convert_and_test(pt_model, (np.random.random((1, 8, 3))), verbose=False)
```
Comment (doronHarnoy) [inline test/layers/elementwise/test_round.py:17]: Really cool test i like it a lot!!

**Review by doronHarnoy — APPROVED** [review]
Cool PR!

File: onnx2kerastl/operation_layers.py (hunk 3 above)
Comment (tomkoren21) [reply onnx2kerastl/operation_layers.py:652]: I did, will fix

File: onnx2kerastl/operation_layers.py (hunk 4 above)
Comment (tomkoren21) [reply onnx2kerastl/operation_layers.py:653]: I don't think these params can ever propogate from pytorch. I suggest that if we'll encounter them we'll solve it then.

**Pull request merged**